test/dlopen/test_dlopen.c

```diff
@@ -68,6 +73,11 @@ int main()
       fprintf(stderr, "ERROR: Test failed to dlopen libnum.so\n");
       return -1;
    }
+    libnum = dlopen(LIB2_NAME, RTLD_NOW);
```

Comment:
this overwrites `libnum` from line 71, where it holds the handle to `libnum.so`, and hence all tests are now carried out on `libnum2.so`. was this intended?

Reply:
Yes the reason was that we wanted to test the NEXT and default test case in dlsym.